x-pack/plugins/stack_connectors/server/connector_types/cases/cases_webhook/service.test.ts

```diff
@@ -474,6 +474,75 @@ describe('Cases webhook service', () => {
       expect(requestMock).not.toHaveBeenCalled();
       expect(res).toBeUndefined();
     });
+    test('properly encodes external system id as string in request body', async () => {
+      requestMock.mockImplementation(() =>
+        createAxiosResponse({
+          data: {
+            id: '1',
+            key: 'CK-1',
+          },
+        })
+      );
+      service = createExternalService(
+        actionId,
+        {
+          config: {
+            ...config,
+            createCommentJson: '{"body":{{{case.comment}}},"id":{{{external.system.id}}}}',
+          },
+          secrets,
+        },
+        logger,
+        configurationUtilities
+      );
+      await service.createComment(commentReq);
+      expect(requestMock).toHaveBeenCalledWith({
+        axios,
+        logger,
+        method: CasesWebhookMethods.POST,
+        configurationUtilities,
+        url: 'https://coolsite.net/issue/1/comment',
+        data: `{"body":"comment","id":"1"}`,
+      });
+    });
+    test('properly encodes external system id as number in request body', async () => {
```

Comment:
it should never be a number because we save it as a string, but lets be thorough 😝